website/docs/d/vzrsfiltatt.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+subcategory: "Contract"
+layout: "aci"
+page_title: "ACI: aci_contract_subject_one_way_filter"
+sidebar_current: "docs-aci-data-source-filter"
+description: |-
+  Data source for ACI Filter
+---
+
+# aci_filter #
+
+Data source for ACI Filter.
+
+NOTE: The aci_contract_subject_one_way_filter should only be used with datasources as it will create conflicts with the aci_contract_subject resource
+
+
+## API Information ##
+
+* `Class` - vzRsFiltAtt
+* `Distinguished Name` - uni/tn-{name}/brc-{name}/subj-{name}/intmnl/rsfiltAtt-{tnVzFilterName}
```

Comment:
Please used different values instead of "name" like(tenant_name, etc).

It is good to have the "GUI Information" next to the "API Information".